mkosi

```diff
@@ -2054,6 +2076,7 @@ def parse_args():
     group.add_argument("--checksum", action='store_true', help='Write SHA256SUMS file')
     group.add_argument("--sign", action='store_true', help='Write and sign SHA256SUMS file')
     group.add_argument("--key", help='GPG key to use for signing')
+    group.add_argument("--bmap", action='store_true', help='Write .bmap file')
```

Comment:
one nitpick, this is pretty brief, could you extend this help text a tiny bit, and say something like "Write block map file (.bmap) for bmaptool usage".

Otherwise it's very unclear what that actually is, but with this change there's "bmaptool" referenced which is much easier googleable...

Reply:
Done, thanks.